lib/index.js

```diff
@@ -36,8 +36,7 @@ internals.Connection.prototype.flush = function() {
 
         try {
 
-            var now = new Date().getTime();
-            this.table.filter(RethinkDB.row('expiresAt').lt(now)).delete().run(this.client);
+            this.table.filter(RethinkDB.row('expiresAt').lt(RethinkDB.now())).delete().run(this.client);
```

Comment:
The tabbing looks odd in the compare, but if you view the file it is tabbed is fine.